templates/base-supa/tests/helpers.ts

```diff
@@ -0,0 +1,118 @@
+import { env } from 'cloudflare:test';
+
+const PROJECT_REF = env.SUPABASE_PROJECT_REF;
+if (!PROJECT_REF) {
+  throw new Error("Missing Environment Variable: SUPABASE_PROJECT_REF");
+}
+
+const BEARER_TOKEN = env.SUPABASE_ANON_KEY;
+if (!BEARER_TOKEN) {
+  throw new Error("Missing Environment Variable: SUPABASE_ANON_KEY");
+}
+
+const BASE_URL = 'https://api.supabase.com/v1';
+
+/**
+ * This file isn't actively used in this template, since Supabase requires
+ * a paid subscription to create branches. It has been included to demonstrate
+ * how to create temporary testing branches via API.
+ * @note To protect user privacy (and minimize branch setup time) it is vital
+ * to branch from a dedicated testing or development database.
+ */
+
+/**
+ * Branch the target database, copying the schema and all current data.
+ * Requires premium Supabase plan.
```

Comment:
```suggestion
 * Requires Supabase "Pro" plan.
```

nit - use the same name they use for their plan